pyproject.toml

```diff
@@ -251,13 +251,17 @@ ignore-decorators = [
 ]
 "examples/*/*.py" = [
     "D205", # 1 blank line required between summary line and description
+    "D400", # First line should end with a period
```

Comment:
I added and exception for D400 in tutorials and examples, as the first line is usually something like:

```
"""
.. _ex-sensitivity-maps:

================================================
Display sensitivity maps for EEG and MEG sensors
================================================
[...]
```